tests/Support/SupportCollectionTest.php

```diff
@@ -5130,6 +5158,129 @@ public function collectionClassProvider()
             [LazyCollection::class],
         ];
     }
+
+    public function testRenameKeys()
```

Comment:
Why is there this test? It doesn't test your new method `orWhere`...

Reply:
hmm... it is committed by mistake. Thanks for mentioning it.